Sources/Subs.php

```diff
@@ -5987,7 +5987,10 @@ function smf_list_timezones($when = 'now')
 		if ($tzid == 'UTC')
 			continue;
 
-		$tz = timezone_open($tzid);
+		$tz = @timezone_open($tzid);
+
+		if ($tz == NULL)
```

Comment:
notice that null is lowercase everywhere; for consistency, can you set this lowercase?